### PR TITLE
Add spending icon

### PR DESCRIPTION
**Pull request by jmophoto**
screen shot 2016-08-18 at 1 32 09 pm

Not to be confused with
search

### CODE REVIEW

File: src/components/Icon.js [hunk 1]
```diff
@@ -1181,6 +1181,24 @@ const Icon = React.createClass({
...
+          	<g>
+          		<g>
+          			<path d='M256,423c-92.1,0-167-74.9-167-167S163.9,89,256,89s167,74.9,167,167S348.1,423,256,423z M256,114.7 c-77.9,0-141.3,63.4-141.3,141.3S178.1,397.3,256,397.3S397.3,333.9,397.3,256S333.9,114.7,256,114.7z'/>
+          		</g>
+          	</g>
+          	<g>
+          		<g>
+          			<path d='M256,320.2c-35.4,0-64.2-28.8-64.2-64.2s28.8-64.2,64.2-64.2s64.2,28.8,64.2,64.2S291.4,320.2,256,320.2z M256,217.5c-21.3,0-38.5,17.3-38.5,38.5s17.3,38.5,38.5,38.5s38.5-17.3,38.5-38.5S277.3,217.5,256,217.5z'/>
+          		</g>
+          	</g>
```
Comment (cerinman) [inline src/components/Icon.js]: weird spacing here

Comment (jmophoto) [inline src/components/Icon.js]: Damn copy and paste.

**Comment by redperk** [comment]
Also not to be confused with Simon:
screen shot 2016-08-18 at 1 44 18 pm

**Comment by redperk** [comment]
:shipit:

**Pull request merged**